pallets/order-book/src/lib.rs

```diff
@@ -516,21 +518,21 @@ impl<T: Config> Pallet<T> {
         )
     }
 
-    // todo: make pub(tests) (k.ivanov)
     /// Validity of asset ids (for example, to have the same base asset
     /// for dex and pair) should be done beforehand
-    pub fn register_tech_account(
+    #[cfg_attr(feature = "test", visibility::make(pub))]
```

Comment:
Could we use something like this?
`#[cfg(test, visibility::make(pub))]`
In this case it is not necessary to use `test` feature falg

Reply:
Sorry, forgot to explain why I did this. Here's the explanation https://github.com/sora-xor/sora2-network/issues/402#issuecomment-1549344136

This approach is a bit ugly, I agree. If it's too bad I think we can leave it as-is and make everything public (features become a bit messy with `test`)